commonpage/search.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://uispark.netlify.app/allComponents.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css "
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="../../style.css" />
+    <title>SearchPage</title>
+  </head>
+  <body>
+    <nav class="flex-H-space-around">
+      <a href="../index.html"><div><span class="span-style">SparkCart</span></div></a>
+      <a href="../../commonpage/search.html"><div>
+        <i class="fas fa-search"></i>
+        Search.....
+      </div></a>
```

Comment:
Suggestion: you can make this a button in future updates
I does not have a border or background so user might get confused,

Else everything is good!
Good to merge!